apps/nextjs-website/src/components/organisms/Auth/EditPasswordForm.tsx

```diff
@@ -46,27 +46,27 @@ export const EditPasswordForm = ({
     }
 
     setErrors(err);
-    const hasErrors = Object.keys(err).length > 0;
-    return !hasErrors;
-  }, [passwords, t]);
+    return Object.keys(err).length === 0;
+  };
 
   const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-    const {
-      target: { value, name },
-    } = event;
-
+    const { value, name } = event.target;
+    setErrors({});
     setPasswords((prev) => ({ ...prev, [name]: value }));
   };
 
   const handleSave = () => {
-    if (!validateForm()) return;
-    onSave(passwords.currentPassword, passwords.newPassword).catch((error) => {
-      if (error.code === 'NotAuthorizedException') {
-        setErrors({ currentPassword: t('changePassword.wrongPassword') });
-      } else {
-        console.error(error);
-      }
-    });
+    if (validateForm()) {
+      onSave(passwords.currentPassword, passwords.newPassword).catch(
+        (error) => {
+          if (error.code === 'NotAuthorizedException') {
+            setErrors({ currentPassword: t('changePassword.wrongPassword') });
+          } else {
+            console.error(error);
```

Comment:
I don't think we need to print the error on console. @MarcoPonchia what do you think about this?

Reply:
Yes i would display instead a generic error (like 404 page)